htdocs/core/class/commonobject.class.php

```diff
@@ -4374,9 +4374,10 @@ public function getSpecialCode($lineid)
 	 *  Check is done into this->childtables. There is no check into llx_element_element.
 	 *
 	 *  @param	int		$id			Force id of object
+	 *  @param	int		$entity		Force entity to check
 	 *  @return	int					<0 if KO, 0 if not used, >0 if already used
 	 */
-	public function isObjectUsed($id = 0)
+	public function isObjectUsed($id = 0, $entity = 0)
```

Comment:
this method is used in object::delete methods
Restrict the search in an entity allows the user to delete an object in use in entity 3 when verifing entity 2 (in which it is not used)

Reply:
@atm-greg @eldy 
"entity" parameter  is used only with Multicompany module !
if this poses a problem, it is only in Multicompany that you will have to see to correct it.